react/features/base/react-native/AudioMode.js

```diff
@@ -0,0 +1,20 @@
+import { NativeModules } from 'react-native';
+import { Platform } from '../react';
+
+let AudioMode;
+
+if (Platform.OS === 'ios') {
+    AudioMode = NativeModules.AudioMode;
+} else {
+    // TODO(saghul): Implement for Android
+    AudioMode = {
+        DEFAULT: 0,
+        AUDIO_CALL: 1,
+        VIDEO_CALL: 2,
+        setMode() {
+            return Promise.resolve(null);
+        }
+    };
+}
```

Comment:
1. I think we can do away with their `Platform.OS` check here. Either `NativeModules.AudioMode` is defined or not. Practically, it's not going to be defined on Android at this time. So with the `Platform.OS` check we're sure to have future work: once we implement it on Android, we'll have to modify this file again.

2. Why put a stub `AudioMode`? I don't think it's better to tell consumers that it's supported given that it's really not.